templates/pioneer-study/pathfinder-log.1.parquetmr.txt

```diff
@@ -0,0 +1,20 @@
+message online-news-log {
```

Comment:
Likewise, please rename to "pathfinder-log"

Reply:
oops! good catch